src/modules/watch/service.test.js

```diff
@@ -8,24 +11,91 @@ describe('test suite for WatchService', () => {
   const mongol = new Mongol(MONGO_URL, 'testdb')
   let db
   let service
+  let watch
+  const objectID = new ObjectID()
```

Comment:
Đặt tên biến là `objectID` khó hiểu lắm, lúc đầu t còn chả biết là cái gì.

```suggestion
  const userID = new ObjectID()
```